common/src/main/java/org/apache/gravitino/audit/FilesetDataOperation.java

```diff
@@ -25,6 +25,10 @@ public enum FilesetDataOperation {
   CREATE,
   /** Opens a file. */
   OPEN,
+  /** Opens a file and writes to it. */
+  OPEN_AND_WRITE,
+  /** Opens a file and appends to it. */
+  OPEN_AND_APPEND,
```

Comment:
Should you also update the Java GVFS code?

Reply:
> Should you also update the Java GVFS code?

Java GVFS does not need to be updated because in the Hadoop FileSystem interface definition, the `open` operation only represents opening a file stream, and there is no situation where one `open` interface has multiple data operations. This situation currently only exists in the `open` interface definition of fsspec.